src/interfaces/IDecryptionSender.sol

```diff
@@ -66,6 +66,30 @@ interface IDecryptionSender {
      * @return Bytes string representing the public key points on the elliptic curve.
      */
     function getPublicKeyBytes() external view returns (bytes memory);
+    
+    /**
+     * @notice Returns all the fulfilled request ids.
+     * @return A uint array representing a set containing all fulfilled request ids.
+     */
+    function getAllFulfilledRequestIds() external view returns (uint256[] memory);
+
+    /**
+     * @notice Returns all the request ids that are yet to be fulfilled.
+     * @return A uint array representing a set containing all request ids that are yet to be fulfilled.
+     */
+    function getAllUnfulfilledRequestIds() external view returns (uint256[] memory);
+
+    /**
+     * @notice Returns count of all the request ids that are fulfilled.
+     * @return A uint representing a count of all request ids that are fulfilled.
+     */
+    function getCountOfFulfilledRequestIds() external view returns (uint256);
```

Comment:
do we have a use case for these? seems like it'd be easy to calculate from the other methods

Reply:
The function has been removed.